app/templates/dashboard/_admin_ccs_data_controller_index.html

```diff
@@ -1,5 +1,5 @@
 {# ADMIN CCS DATA CONTROLLER #}
-<h2 class="heading-xmedium">Search for users</h3>
+<h2 class="govuk-heading-m">Search for users</h3>
```

Comment:
It would be great if we could split these changes to remove `heading-*` classes into a separate commit or PR, so that we can translate those changes into a PR that other devs on the team can do.

Reply:
I think it's going to be difficult to split it out because these classes can be inconsistent and it depends on the context that they being used whether the heading size is appropriate.